nixos/modules/services/torrent/transmission.nix

```diff
@@ -21,6 +21,16 @@ let
 
   # for users in group "transmission" to have access to torrents
   fullSettings = { umask = 2; download-dir = downloadDir; incomplete-dir = incompleteDir; } // cfg.settings;
+
+  preStart = pkgs.writeScript "transmission-pre-start" ''
+#!${pkgs.runtimeShell}
```

Comment:
Could you indent this? `''` strings will correctly remove the trailing indentation.